api/definition/indicators/paths.yaml

```
@@ -138,11 +138,13 @@ indicators_by_indicator_id_all_data:
     - $ref: "./parameters.yaml#/inequalities"
     - $ref: "./parameters.yaml#/latest_only"
   get:
+    security:
+      - bearerAuth: [ ]
     tags:
       - indicators
     summary: Get published and unpublished health data for an indicator
     description:
-      Get data for a public health indicator. This will return all data for
+      Authorised user can Get data for a public health indicator. This will return all data for
```

I don't think you need to add the `Authorised user` bit. I would say the `security` section covers that authentication/authorisation is required.
```suggestion
      Get data for a public health indicator. This will return all data for
```